docker-vertica/Dockerfile

```
@@ -65,18 +67,20 @@ RUN set -x \
 # Copy in a stable ssh key.  This is done so that Vertica pods can communicate
 # with pods running an older image.  This is necessary when doing an online
 # image change as the Vertica cluster will be running with two container
-# versions at once.
+# versions at once. This step is required if not including SSH keys because
+# conditional copy isn't a thing in Docker. If no SSH keys are to be included, 
+# then the next RUN will remove the key we just added.
 COPY dbadmin/.ssh /home/dbadmin/.ssh
 
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 RUN set -x \
-  && mkdir -p /root/.ssh \
-  && cp -r /home/dbadmin/.ssh /root \
-  && chmod 700 /root/.ssh \
-  && chmod 600 /root/.ssh/* \
   && chmod 700 /home/dbadmin/.ssh \
   && chmod 600 /home/dbadmin/.ssh/* \
   && chown -R dbadmin:verticadba /home/dbadmin/ \
-  && chmod go-w /etc/ssh/sshd_config.d/* /etc/ssh/ssh_config.d/*
+  && chmod go-w /etc/ssh/sshd_config.d/* /etc/ssh/ssh_config.d/* \
+  && if [[ ($NO_SSH_KEYS == "YES" || $NO_SSH_KEYS == "yes") ]] ; then \
```

Can we simplify this to do a case-insensitive compare?
```suggestion
  && if [[ ${NO_SSH_KEYS^^} == "YES" ]] ; then \
```

---

Ok. I can also do the same with `MINIMAL`.

---

Yes, that would be good thanks.